PSL/SENSORS/VL531X.py

```diff
@@ -0,0 +1,451 @@
+# https://www.st.com/resource/en/datasheet/vl53l0x.pdf
+# https://www.st.com/resource/en/datasheet/vl53l1x.pdf
+
+# see also:
+# https://github.com/adafruit/Adafruit_VL53L0X/blob/master/src/vl53l0x_tuning.h#L49
+# https://github.com/pololu/vl53l0x-arduino/blob/master/VL53L0X.cpp
+# https://os.mbed.com/teams/ST/code/VL53L0X//annotate/834986cdde0a/VL53L0X_tuning.h/
+# https://git.nukon.com.au/public-group/iot-counter/blob/124e2d2f6a5690ec11c67aaa5b70bb8cca27bca8/VL50L0X_library/Api/core/inc/vl53l0x_tuning.h
+# https://github.com/adafruit/Adafruit_CircuitPython_VL53L0X/blob/master/adafruit_vl53l0x.py
+
+from __future__ import print_function
+import time
+
+from PSL.Peripherals import I2CSlave
+
+def connect(route, **args):
```

Comment:
What is the purpose of this function? Why not have the user call VL531X() directly?